.github/workflows/release.yml

```diff
@@ -5,20 +5,13 @@ name: release
 on:
   workflow_dispatch:
     inputs:
-      branch:
-        description: 'The branch to release'
+      ref:
+        description: 'Branch or tag ref to run the workflow on'
         required: true
-        default: 'main'
+        default: "main"
       version:
         description: 'The version to release (e.g. 1.2.3). This workflow will automatically perform the required version bumps'
         required: true
-      update_changelog:
```

Comment:
What's the reason to remove this flag?

Reply:
Before the process was not idempotent and was pushing the changelog directly in the branch.
This means that the second run would fail.
With the PR approach, you can close the PR and rerun the job itself.
The flag isn't require anymore.